sdk/metricsadvisor/Azure.AI.MetricsAdvisor/src/MetricsAdvisorClient.cs

```diff
@@ -106,6 +68,7 @@ protected MetricsAdvisorClient()
         /// <returns>An <see cref="AsyncPageable{T}"/> containing the collection of values the specified <see cref="DataFeedDimension"/> can assume.</returns>
         /// <exception cref="ArgumentNullException"><paramref name="metricId"/> or <paramref name="dimensionName"/> is null.</exception>
         /// <exception cref="ArgumentException"><paramref name="metricId"/> or <paramref name="dimensionName"/> is empty; or <paramref name="metricId"/> is not a valid GUID.</exception>
+        [ForwardsClientCalls]
```

Comment:
Can we remove this once https://github.com/azure/autorest.csharp/issues/2069 is complete?

Reply:
Issue tracking here: https://github.com/Azure/azure-sdk-for-net/issues/28124

Reply:
After https://github.com/Azure/azure-sdk-for-net/pull/28353 updates. I've removed `ForwardsClientCalls` attribute.